superset/databases/api.py

```diff
@@ -265,3 +275,87 @@ def select_star(
             return self.response(404, message="Table not found on the database")
         self.incr_stats("success", self.select_star.__name__)
         return self.response(200, result=result)
+
+    @expose("/schemas/", methods=["GET"])
+    @protect()
+    @safe
+    @statsd_metrics
+    @rison(get_schemas_schema)
+    def schemas(self, **kwargs: Any) -> FlaskResponse:
+        """Get all schemas
+        ---
+        get:
+          parameters:
+          - in: query
+            name: q
+            content:
+              application/json:
+                schema:
+                  type: object
+                  properties:
+                    page_size:
+                      type: integer
+                    page:
+                      type: integer
+                    filter:
+                      type: string
```

Comment:
It's nicer to use a reference here to the schema, so we don't have to repeat ourselfs. Take a look at: https://github.com/apache/incubator-superset/blob/master/superset/charts/api.py#L502